.github/ISSUE_TEMPLATE/new-idea.md

```diff
@@ -6,15 +6,15 @@ about: Template for a new project idea. Remove comments before posting.
 
 ## Project description
 
-[_Describe the project the best you can. Give any background information or link to resources that are necessary to understand the problem it is intended to solve. The more you elaborate on your idea, the easier it is to accomplish._]
+<!--[_Describe the project the best you can. Give any background information or link to resources that are necessary to understand the problem it is intended to solve. The more you elaborate on your idea, the easier it is to accomplish._]-->
```

Comment:
It may make sense to remove `[_` and `_]` too. Please rewrite the history (in order not to produce unneeded commits) if you prefer to do it.